src/datagen/java/blusunrize/immersiveengineering/data/recipes/builder/MineralMixBuilder.java

```diff
@@ -45,9 +47,21 @@ public static MineralMixBuilder builder()
 		return new MineralMixBuilder();
 	}
 
-	public MineralMixBuilder dimension(ResourceKey<DimensionType> dimension)
+	public MineralMixBuilder biomeCondition(Set<TagKey<Biome>> tags)
```

Comment:
Varargs (`TagKey<Biome>...`)? Probably needs `@SafeVarargs`, but that should be fine here. That would get rid of the `(Immutable)Set.of` at the calls.

Reply:
Yeah the lack of varargs safety annoyed me which is why I did Set :'D